tests/analysis/damages/test_damages_utils.py

```diff
@@ -64,14 +64,34 @@ def test_clean_lane_data(self):
         assert _result_data[0] == 42
         assert _result_data[1] == 24
 
-    def test_create_summary_statistics(self):
+    @pytest.mark.parametrize(
+        "lanes, expected",
+        [
+            pytest.param([0, 1, 3], [0, 1, 3], id="Valid lanes"),
+            pytest.param([np.nan, 1, 3], [2, 1, 3], id="First lane invalid"),
+            pytest.param([0, np.nan, 3], [0, 1.5, 3], id="Middle lane invalid"),
+            pytest.param([0, 1, np.nan], [0, 1, 0.5], id="Last lane invalid"),
+            pytest.param(
+                [np.nan, np.nan, np.nan],
+                [np.nan, np.nan, np.nan],
+                id="All lanes invalid",
+            ),
+        ],
+    )
+    def test_create_summary_statistics(self, lanes, expected):
+        def valid_lanes(lane: float, expected: float) -> bool:
+            if np.isnan(lane):
+                return np.isnan(expected)
+            return lane == pytest.approx(expected)
```

Comment:
please move to the validation part, is a bit misleading having it before defining test data.

Reply:
Adjusted